### ISSUE TITLE
sphinx docs

### ISSUE THREAD

**Issue by oz123** [mgorny/portage-mgorny#20]
Hi,

I really appreciate the effort. I was disillusioned very quickly from my thoughts of contributing to portage, since the code is really hard to follow. There seems to be now coding standard (at least no pep8) and there are no docs.

Would you accept a PR for sphinx docs?

**Comment by mgorny** [comment]
Surely I wouldn't mind. However, you may want to look into providing something for [upstream Portage](https://github.com/gentoo/portage). It still has some epydoc tags all around the place (whose support I've removed because epydoc is long dead), and some old docbook docs (which didn't seem worthy of much attention).

**Comment by oz123** [comment]
As your readme says, it's quite hard to contribute upstream ...
I am also not officially a gentoo member.
